README.md

```diff
@@ -49,8 +43,6 @@ In this shield study, we have two experimental branches based on notification in
 2. notification-interval-longer - Cohort that’s shown notification in download panel after 2 days interval (since last notification shown and dismissed by clicking button Not Now)
 3. notification-interval-short - Cohort that’s shown notification after shorter interval - 1 day
 
-To force override a variation, set preference 'cloud.services.shield.variation' with the variation name
```

Comment:
You may re-add this, but phrased as "To force override a variation, clone the repo and set `testing.variationName` in `studySetup.js` to the variation name"